.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish
+on:
+  push:
+    branches:
+      - 'main'
+      - 'master'
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: "0.81.0"
+
+      - name: Build Hugo Site
+        run: hugo --minify
+
+      - name: Deploy GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          personal_token: ${{ secrets.PUBLISH_TOKEN }}
+          publish_dir: ./public
+          publish_branch: gh-pages
+          cname: rotational.io
```

Comment:
@rebeccabilbro I know you tried something similar in a previous commit then removed it; I had been using Travis for deployment in my personal blog but wanted to check out GitHub actions, so I brought it back.